### ISSUE TITLE
Flashing issue when resizing QtInteractor.

### ISSUE THREAD

**Issue by JacobBumgarner** [pyvista/pyvistaqt#102]
### **Epilepsy warning**

When resizing a pyqt5 window, the QtInteractor window flashes.

https://user-images.githubusercontent.com/70919881/116622816-47dbec00-a913-11eb-8f78-e5bb370dfe92.mov

**Comment by JBum21** [comment]
Similarly to #101, this seems to be due to issue with PyQt versions 5.14 >=

Not sure if I should mark this as closed or not... thoughts?

**Comment by GuillaumeFavelier** [comment]
Hello @JBum21, thanks for reporting, I was not aware of this one.

> this seems to be due to issue with PyQt versions 5.14 >=

How did you figure out?

> Not sure if I should mark this as closed or not... thoughts?

Could you share your report before closing? It would be helpful for future reference, thank you.

```
python -c `import pyvista; print(pyvista.Report)`
```

**Comment by GuillaumeFavelier** [comment]
Actually, let's leave it open similarly to #20 since it's more a workaround than a real fix.

**Comment by JBum21** [comment]
Sounds good. Figured this out due to experimentation with #101.

@GuillaumeFavelier 
Since I have you here at the moment, can you help me with another question I have? Why is QtInteractor so much slower than the plotter? 
Rendering a mesh using plotter takes around 10 seconds, whereas the QtInteractor can take north of 60s to unfreeze after adding the mesh. Is this just a known effect of integration into PyQt5 that VTKRenderWindowInteractor? I can provide more details if needed.

Not sure if this is really a question worth opening an issue for.

**Comment by GuillaumeFavelier** [comment]
This is not something that I investigated because I did not notice such a big difference. `60s` instead of `10s` is huge! Could you give more details about your benchmark in a different issue? It would be very convenient if you could produce a minimum working example to reproduce :)

> unfreeze after adding the mesh

Hm... I guess it could be a delay in `paintEvent` or `updateEvent`.

**Comment by JBum21** [comment]
Yes I will work to get a reproducible build for you. The plotter vs. interactive load times become drastically more apparent when rendering meshes that take > 60s on plotter. 

When loading glyphed meshes, I experience no difference between plotter/Interactor. However, my issue comes when I add the Multiblocks I’m using represent hundreds of spline.tubes; maybe it’s just an issue with my implementation of pyvista.

Thanks for being so helpful. This is an amazing library with such supportive devs. Will @ you when I have a build & some logs up. Cheers

**Comment by JBum21** [comment]
@GuillaumeFavelier  Ok, I dramatized the 10s vs 60s. It's more like 10s vs 40s. I've further discovered that this issue seems to exclusively occur with .vtmb files, not .vtk files.


Please see the code here for event reproduction:
```
import pyvista as pv
from pyvistaqt import BackgroundPlotter

# Loading my network vtmb file here.
network = pv.read("network.vtmb")

# First add to plotter.
p = pv.Plotter()
p.add_mesh(network, scalars='Length (µm)', smooth_shading=True, pickable=False)
p.show()
p.close()

# Then add to background plotter after closing pv.plotter. Notice lag.
bp = BackgroundPlotter()
bp.add_mesh(network, scalars='Length (µm)', smooth_shading=True, pickable=False)
bp.app.exec()
```

And please see [this link](https://github.com/JBum21/Plotter-vs-Background) to download my example file.
I've excluded tube ends (the file is too big, ~100mb), which I separately glyph endpoints as spheres.  If I just add the tube caps separately, I experience no difference between plotter and background load times.

See an example image of what the mesh looks like with caps below.


![CapsOn](https://user-images.githubusercontent.com/70919881/117463387-98f46d00-af1d-11eb-90a6-29c6f39c5d2f.png)
